src/applications/vaos/utils/eligibility.js

```diff
@@ -100,9 +113,9 @@ function hasVisitedInPastMonthsDirect(eligibilityData) {
 
 function hasVisitedInPastMonthsRequest(eligibilityData) {
   return (
-    eligibilityData.requestPastVisit.durationInMonths ===
+    eligibilityData.requestPastVisit?.durationInMonths ===
```

Comment:
Adding `recordEligibilityGAEvents()` to the `newAppointment` actions was causing some tests in `newAppointment.unit.spec.js` to fail since `eligibilityData` is null in many of these tests so I added optional chaining to some of these checks.